### PR TITLE
add: scraper support for www.dr.dk

### PR DESCRIPTION
**Pull request by MarcStorm**
Solves #292 

Encoding added to support Danish chars, 'ø', 'æ' and 'å'.

### CODE REVIEW

File: recipe_scrapers/__init__.py [hunk 1]
```diff
@@ -130,6 +131,10 @@
 from .wikicookbook import WikiCookbook
 from .yummly import Yummly
 
+import sys
+
+sys.stdout.reconfigure(encoding="utf-8")
```
Comment (MarcStorm) [inline recipe_scrapers/__init__.py:136]: This doesn't appear to be supported in >v. 3.7

Comment (hhursev) [reply recipe_scrapers/__init__.py:136]: I need to educate myself on this topic and will get back to you.

I don't have problems with Danish or similar chars, even if I don't add this line. I am with utf-8 stdout encoding by default. Can you tell me what is the value of your PYTHONIOENCODING env var:

```
import os
os.environ.get('PYTHONIOENCODING')
```

aside from these couple lines it looks perfect for me and I'd like to merge and bump version 🍻

Comment (MarcStorm) [reply recipe_scrapers/__init__.py:136]: Result: `none`
Ran `export PYTHONIOENCODING=utf-8` and it works like a charm, thus has the marked lines been removed.

**Comment by hhursev** [comment]
Merging and bumping version 🍻  😉

**Pull request merged**